docs/mkdocs.yml

```diff
@@ -40,6 +40,8 @@ nav:
       - Migration Troubleshooting: guides/migration-troubleshooting.md
       - Terms of Use Setup: guides/providing-terms.md
   - Configuration: configuration.md
+  - References:
+    - 'HedgeDoc Flavored Markdown': references/hfm.md
```

Comment:
Do we want to put the page in it's own category in the old page, too? I did so in the new one, because I intended to include other references in this category, too.

Reply:
Where else do you want to put it?

Reply:
That's a good question and I'm not to sure. It's not really a guide and it's not developer related. It's also not concerned with installation or configuration. This is probably the right place to put it.